proto/api/api.proto

```diff
@@ -193,6 +193,10 @@ message BlockInfo {
   iotextypes.TransactionLogs transactionLogs = 3;
 }
 
+message BlockInfos {
```

Comment:
if your using this in API, this is fine, but if you plan to use this in db, probably put somewhere else

Reply:
yeah, let me move it